receiver/windowsperfcountersreceiver/config.go

```diff
@@ -46,23 +72,73 @@ func (c *Config) Validate() error {
 		errs = multierr.Append(errs, fmt.Errorf("must specify at least one perf counter"))
 	}
 
+	if len(c.MetricMetaData) == 0 {
+		errs = multierr.Append(errs, fmt.Errorf("must specify at least one metric"))
+	}
+
+	for _, metric := range c.MetricMetaData {
+		if metric.MetricName == "" {
+			errs = multierr.Append(errs, fmt.Errorf("a metric does not include a name"))
+			continue
+		}
+		if metric.Description == "" {
+			errs = multierr.Append(errs, fmt.Errorf("metric %q does not include a description", metric.MetricName))
+		}
+		if metric.Unit == "" {
+			errs = multierr.Append(errs, fmt.Errorf("metric %q does not include a unit", metric.MetricName))
+		}
+
+		if (metric.Gauge == GaugeMetric{}) && (metric.Sum == SumMetric{}) {
+			errs = multierr.Append(errs, fmt.Errorf("metric %q does not include a metric definition", metric.MetricName))
+		} else if (metric.Gauge != GaugeMetric{}) {
+			if metric.Gauge.ValueType == "" {
+				errs = multierr.Append(errs, fmt.Errorf("gauge metric %q does not include a value type", metric.MetricName))
+			} else if metric.Gauge.ValueType != "int" && metric.Gauge.ValueType != "double" {
+				errs = multierr.Append(errs, fmt.Errorf("gauge metric %q includes an invalid value type", metric.MetricName))
+			}
+		} else if (metric.Sum != SumMetric{}) {
```

Comment:
I think we need a check for if they specify both `gauge` and `sum`

Reply:
added although this might become irrelevant based on the above implementation

Reply:
As stated above, I think defaulting to `gauge` is reasonable, but we should still check if they've specified both.

Reply:
I more meant the implementation of the data type field because there's nothing to be filled out in the gauge struct since there is no value type